core/src/main/java/com/linecorp/armeria/client/DnsResolverGroupBuilder.java

```diff
@@ -292,6 +295,16 @@ public DnsResolverGroupBuilder decodeIdn(boolean decodeIdn) {
         return this;
     }
 
+    /**
+     * Sets MeterRegistry.
+     * @param meterRegistry {@link MeterRegistry}.
+     * @return DnsResolverGroupBuilder.
```

Comment:
Let's remove this return statement and just do:
`Sets {@link MeterRegistry} to collect the DNS query metrics.`

Reply:
done